public/layout.css

```diff
@@ -11,12 +11,14 @@ html, body, uiview {
 }
 
 .container {
-	min-height: 100vh;
+	height: 100vh;
+	overflow: auto;
```

Comment:
The problem is created when `.container` is taller than the viewport, thus the reason for horizontal scroll on the sheet selection screen.  Giving `.container` a height that will not exede the viewport fixes this bug.